exercise-core/src/main/java/com/examples/attsw/exercise/core/controller/EmployeeController.java

```diff
@@ -0,0 +1,26 @@
+package com.examples.attsw.exercise.core.controller;
+
+import com.examples.attsw.exercise.core.model.Employee;
+import com.examples.attsw.exercise.core.service.IEmployeeService;
+
+public class EmployeeController {
+
+	private IEmployeeService iEmployeeService;
+
+	public EmployeeController(IEmployeeService iEmployeeService) {
+		this.iEmployeeService = iEmployeeService;
+	}
+
+	public String getAllEmployee() {
+		String stringEmployee = "";
+		for (Employee employee : iEmployeeService.getEmployees()) {
+			stringEmployee += employee.toString();
```

Comment:
the issue required that each employee should be in a different line... this does not add any end of line...